system/nxplayer/nxplayer.c

```diff
@@ -148,7 +148,7 @@ static int _open_with_http(const char *fullurl)
   char resp_msg[] = "\r\n\r\n";
   struct timeval tv;
   uint16_t port = 80;
-  char buf[64];
+  char buf[256];
```

Comment:
why this is 256? Ia it somehow depending on `CONFIG_NXPLAYER_HTTP_MAXFILENAME`?

Reply:
Yes, but MAX_PATH is safe value, since the format string add more char to relurl:
```
  snprintf(buf, sizeof(buf), "GET /%s HTTP/1.0\r\n\r\n", relurl);
```
it's hard and unnecessary to calculate the real size.